pkg/xray/resource_xray_repository_config.go

```diff
@@ -0,0 +1,349 @@
+package xray
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"github.com/jfrog/terraform-provider-shared/validator"
+)
+
+type GeneralRepoConfig struct {
```

Comment:
Same with the attribute, let's remove the `Repo` prefix from the type name. So `Configuration` suffice.

Reply:
done.